modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_uploader/js/utils/sendFile.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const noop = () => {};
+
+function parse(req) {
+	var result;
```

Comment:
I see you're using `var` here but you're using `let` and `const` in most places, is this intentional?

Reply:
No, it's a mistake. I will use let. ty!